packages/localizations/src/pt-BR.ts

```diff
@@ -354,7 +358,7 @@ export const ptBR: LocalizationResource = {
     noAvailableMethods: {
       message: 'Não foi possível fazer login. Não há nenhum método de autenticação disponível.',
       subtitle: 'Aconteceu um erro',
-      title: 'Não foi possível fazer login',
+      title: 'Não foi possível entrar',
```

Comment:
Just to clarify the change here - why is the reason of replacing "fazer login" with "entrar"?

Reply:
I thought I reverted this change, sorry!
Realized fazer login is the nomenclature everywhere